sdk/communication/azure-communication-identity/tests/acs_identity_test_case.py

```diff
@@ -52,3 +59,28 @@ def generate_teams_user_aad_token(self):
 
     def skip_get_token_for_teams_user_test(self):
         return str(self.skip_get_token_for_teams_user_tests).lower() == 'true'
+
+    def get_async_client(self):
```

Comment:
```suggestion
    def create_client_from_connection_string(self):
```
wdyt about renaming and putting it into the `test_communication_identity_client_async.py`, as it's used only there?

Reply:
Done